apps/ledger-live-desktop/tests/specs/general/keyboard-shortcuts.spec.ts

```diff
@@ -0,0 +1,67 @@
+import os from "os";
+import test from "../../fixtures/common";
+import { expect } from "@playwright/test";
+import { Layout } from "../../models/Layout";
+import { AddAccountModal } from "tests/models/AddAccountModal";
+import { PortfolioPage } from "tests/models/PortfolioPage";
+
+test.use({ userdata: "skip-onboarding" });
+
+// eslint-disable-next-line jest/expect-expect
```

Comment:
Is this really needed ?

Reply:
Yes I test keyboard navigation in modals, so I need to skip onboarding.